README.md

```diff
@@ -28,10 +31,20 @@ You can create RabbitMQ resources:
 6. [Policy](./docs/examples/policies)
 7. [Permissions](./docs/examples/permissions)
 
-## Install without cert-manager
+## Documentation
 
-If you do not have cert-manager installed in your k8s cluster, you will need to generate certificates used by admission webhooks yourself and include them in the operator and webhooks manifests.
-You can follow [this doc](./docs/installation/install-without-certmanager.md).
+Messaging Topology Operator is covered in several guides:
+
+ - [Operator overview](https://www.rabbitmq.com/kubernetes/operator/operator-overview.html#topology-operator)
+ - [Installation](https://www.rabbitmq.com/kubernetes/operator/install-topology-operator.html)
+ - [Using Messaging Topology Operator](https://www.rabbitmq.com/kubernetes/operator/using-topology-operator.html)
+ - [TLS](https://www.rabbitmq.com/kubernetes/operator/tls-topology-operator.html)
+ - [Troubleshooting Messaging Topology Operator](https://www.rabbitmq.com/kubernetes/operator/troubleshooting-topology-operator.html)
+
+In addition, a number of [examples](./docs/examples) can be found in this repository.
+
+The doc guides are open source. The source can be found in the [RabbitMQ website repository](https://github.com/rabbitmq/rabbitmq-website/)
+under `site/kubernetes`.
 
 ## RabbitMQCluster requirements
```

Comment:
This section still refers to the old TLS guide that is now on rabbitmq.com, so it will be a broken link.